### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by githubjoiner** [cheapthrillandwine/Improving_Unsupervised_Defect_Segmentation#1]
why the loss function in your code is different from your paper when training the network.

**Comment by love6tao** [comment]
@githubjoiner  the loss fuction in the code is binary_crossentropy, not ssim loss. the result images shown in the code look good. so this code just uses the encoder-decoder network in the paper

**Comment by love6tao** [comment]
the code of AE has an error. "encoded = Conv2D(1, (8, 8), strides=1, padding='same')(x)"  the filter number shoud be 100, not 1

**Comment by nuclearboy95** [comment]
this is not an SSIM-AE,,,

**Comment by mozhuqing** [comment]
> this is not an SSIM-AE,,,
Have you reproduced it? I've been watching this lately. Please.

**Comment by mozhuqing** [comment]
> the code of AE has an error. "encoded = Conv2D(1, (8, 8), strides=1, padding='same')(x)" the filter number shoud be 100, not 1
Have you reproduced it? I've been watching this lately. Please.

**Comment by mozhuqing** [comment]
> @githubjoiner the loss fuction in the code is binary_crossentropy, not ssim loss. the result images shown in the code look good. so this code just uses the encoder-decoder network in the paper

Have you reproduced it? I've been watching this lately. Please.
